orchagent/response_publisher_interface.h

```diff
@@ -5,32 +5,31 @@
 #include "return_code.h"
 #include "table.h"
 
-class ResponsePublisherInterface {
- public:
-  virtual ~ResponsePublisherInterface() = default;
+class ResponsePublisherInterface
```

Comment:
looks like this file is not changed but added with some tabspace? Do you want to revert this?

Reply:
We ran clang-format using Microsoft style. This matches other p4orch codes and the SONiC codes.
I think we should make this change to keep the style consistent.
I can create a different PR for this if you need. Otherwise, we can keep the change here.